newsletter/2024-06-25-these-weeks-in-loongarch-52.md

```diff
@@ -0,0 +1,90 @@
+---
+slug: these-weeks-in-loongarch-52
+title: 每周两龙：第 52 期
+authors: [xen0n]
+date: 2024-06-25T23:00:00+08:00
+tags: [每周一龙]
+draft: true  # TODO
+---
+
+每周（尽量）为大家报道 LoongArch 社区最前线的第一手新鲜资讯！
+上周的进展主要有 TODO。
+详情点进来看！
+
+<!-- truncate -->
+
+如无特别说明，文中提及的日期、时间都为北京时间（UTC+8）。
+
+## 先「马」再看 {#marked-projects}
+
+本栏目的内容具有一定延续性，将持续追踪报道 LoongArch 领域的重要或长期项目（坑）。
+
+### Linux {#linux}
+
+TODO: amdgpu & LS7A drama [Icenowy]
+
+TODO: [xry111]'s fstat work
```

Comment:
The final result is https://git.kernel.org/pub/scm/linux/kernel/git/vfs/vfs.git/commit/?h=vfs.misc&id=0ef625bba6fb2bc0c8ed2aab9524fdf423f67dd5.

Reply:
I've provided this as a part of a PR against your personal repo.